src/legacy/ui/public/saved_objects/components/saved_object_save_modal.js

```diff
@@ -166,74 +166,77 @@ export class SavedObjectSaveModal extends React.Component {
+
+                <EuiFormRow
+                  label={(<FormattedMessage
+                    id="common.ui.savedObjects.saveModal.titleLabel"
+                    defaultMessage="Title"
+                  />)}
+                >
+                  <EuiFieldText
+                    autoFocus
+                    data-test-subj="savedObjectTitle"
+                    value={this.state.title}
+                    onChange={this.onTitleChange}
+                    isInvalid={this.state.hasTitleDuplicate || this.state.title.length === 0}
+                  />
+                </EuiFormRow>
 
-            {this.renderDuplicateTitleCallout()}
+                {this.props.options}
 
-            <EuiForm>
+              </EuiForm>
 
-              {this.renderCopyOnSave()}
+            </EuiModalBody>
 
-              <EuiFormRow
-                label={(<FormattedMessage
-                  id="common.ui.savedObjects.saveModal.titleLabel"
-                  defaultMessage="Title"
-                />)}
+            <EuiModalFooter>
+              <EuiButton
+                data-test-subj="saveCancelButton"
+                onClick={this.props.onClose}
               >
-                <EuiFieldText
-                  autoFocus
-                  data-test-subj="savedObjectTitle"
-                  value={this.state.title}
-                  onChange={this.onTitleChange}
-                  isInvalid={this.state.hasTitleDuplicate || this.state.title.length === 0}
+                <FormattedMessage
+                  id="common.ui.savedObjects.saveModal.cancelButtonLabel"
+                  defaultMessage="Cancel"
                 />
-              </EuiFormRow>
-
-              {this.props.options}
-
-            </EuiForm>
-
-          </EuiModalBody>
-
-          <EuiModalFooter>
-            <EuiButton
-              data-test-subj="saveCancelButton"
-              onClick={this.props.onClose}
-            >
-              <FormattedMessage
-                id="common.ui.savedObjects.saveModal.cancelButtonLabel"
-                defaultMessage="Cancel"
-              />
-            </EuiButton>
-
-            <EuiButton
-              fill
-              data-test-subj="confirmSaveSavedObjectButton"
-              onClick={this.saveSavedObject}
-              isLoading={this.state.isLoading}
-              isDisabled={this.state.title.length === 0}
-            >
-              {this.props.confirmButtonLabel
-                ? this.props.confirmButtonLabel
-                : (
-                  <FormattedMessage
-                    id="common.ui.savedObjects.saveModal.confirmSaveButtonLabel"
-                    defaultMessage="Confirm Save"
-                  />
-                )
-              }
-            </EuiButton>
-          </EuiModalFooter>
+              </EuiButton>
+
+              <EuiButton
+                fill
+                data-test-subj="confirmSaveSavedObjectButton"
+                onClick={this.saveSavedObject}
```

Comment:
This `onClick` listener should instead be moved to the `onSubmit` of the `form` element.

Reply:
ok!